CHANGELOG.md

```diff
@@ -12,6 +12,8 @@ Please avoid adding duplicate information across this changelog and JIRA/doc inp
 ### Removed Features
 
 ### Deprecated Features
+- ROX-12620: We continue to simplify access control management by grouping some permissions in permission sets. As a result:
+  - The permission `WorkflowAdministration` will deprecate the permissions `Policy, VulnerabilityReports`.
```

Comment:
We haven't done this before but we should have been, so let's start now: provide mitigation strategy for users. Basically, what does this deprecation mean and how should users react? I guess we want to say that users either:
* Should update all their custom permission sets to specify the desired value for `WorkflowAdministration` (and maybe even remove `Policy` and `VulnerabilityReports`, iirc replacing resources logic allows that).
* Wait for 3.76 and have a migration to do the replacement, explaining how this migration will likely work (and including this description into https://issues.redhat.com/browse/ROX-13888). Specifically, if you plan to migrate using `OR`, it makes sense to explicitly say that a permission set with, say, `WRITE Policy` and `READ VulnerabilityReports`, after 3.76 migration this permission set will lose the ability to modify policies.

I guess it makes sense to add the mitigation strategy to changelog and call it out (i.e., mention its existence) in the UI and 3.74.0 release intro doc but I think it makes sense to check with @kcarmichael08 and @vjwilson.

Reply:
@rukletsov Are you thinking of something along the same lines as what we did in `3.73` in the Access Control section,
but with the addition of a link to the 3.74 release notes for a fuller description of these new deprecations/mitigations?

<img width="1404" alt="Screen Shot 2022-12-13 at 9 35 38 AM" src="https://user-images.githubusercontent.com/715729/207362107-794df16c-c088-4d8e-ae4b-bbe57e594910.png">

Reply:
Yes, I'm saying that this banner that is also update in this PR is not enough and I suggest to add a hint there to where (changelog?) a user can read more on what they should do with this.

Reply:
What I've done for the moment is the following:
- A new section within the changelog that specifically outlines potential actions customers have to do as preparations for deprecation. It also notes the strategy we are going to follow within the migration (updated [ROX-13888](https://issues.redhat.com/browse/ROX-13888) as well).
- Create a reference in the form of "...please consolidate the release notes." within the UI banner. I converged on this with @kcarmichael08.

The new section within the changelog will be added within the release notes, this should at the very least make it more clear _what_ this means for customers and steps they can follow preemptively to avoid running into the unwanted side-effects.

Reply:
a. Are we not planning on adding migration for when `Policy` and  `VulnerabilityReports` sac resources are removed?
b. If are adding migration, then users do not have to take any action?

Reply:
We are planning to have a migration, but customers will have to re-review the permission sets that will be changed within the migration.

For all migrations related to resource deprecations, we have followed the following principle: the least access rights of the removed resources will be applied to the migrated resource, i.e. having `READ Policy and WRITE VulnerabilityReports` will become `READ WorkflowAdministration` after the migration.
The action users have to do is to re-visit their permission set if they have the appropriate access rights for everything.

Reply:
👍